clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/caching/KubernetesV2Provider.java

```diff
@@ -42,19 +42,19 @@
     return Collections.newSetFromMap(new ConcurrentHashMap<>());
   }
 
-  public void addAllAgents(Collection<Agent> agents) {
+  public void stageAllAgents(Collection<Agent> agents) {
     nextAgentSet.addAll(agents);
   }
 
-  public void clearNewAgentSet() {
+  public void clearStagedAgents() {
     nextAgentSet.clear();
   }
 
-  public void switchToNewAgents() {
+  public void addStagedAgents() {
     Collection<Agent> nextAgentSetCopy = emptyAgentCollection();
     nextAgentSetCopy.addAll(nextAgentSet);
-    agents = nextAgentSetCopy;
-    clearNewAgentSet();
+    agents.addAll(nextAgentSetCopy);
+    clearStagedAgents();
```

Comment:
Do we need the `nextAgentSetCopy` variable anymore?  Isn't this equivalent to:
```
agents.addAll(nextAgentSet);
clearStagedAgents();
```
Or am I missing something?

Reply:
You're right, we don't need the copy anymore